test-integration/helpers/ui/test-element.coffee

```diff
@@ -94,10 +97,12 @@ class TestHelper extends TestItemHelper
       defaultWaitTime: 40 * 1000 # TODO: Letting tests define their own wait time is dangerous. tutor-dev takes > 10sec to delete a task-plan
       # defaultWaitTime: 20 * 1000 # 20sec seems to be enough for deployed code but not local
 
+    @test = test
     @_options = _.assign {}, defaultOptions, options
     @_el = {}
 
     commonElements.loadingState = @options.loadingLocator
+    commonElements.self = testElementLocator
```

Comment:
[`waitUntilLoaded`](https://github.com/openstax/tutor-js/pull/1000/files#diff-8644634b42f6a903538586ecef0c1451L105) uses `loadingState`.  `self` isn't really right now, but the idea was for `waitUntilLoaded` to check for it's presence as well